README.md

```diff
@@ -125,3 +125,26 @@ app.register(explain, {
     context.reply.request.headers['explain']
 })
 ```
+
+## Add the plugin to mercurius
+
+In the mercurius configuration it is possibile to add the graphiql plugin (mercurius-explain-graphiql-plugin)[https://github.com/nearform/mercurius-explain-graphiql-plugin]
+
+```js
+app.register(mercurius, {
+  schema,
+  resolvers,
+  graphiql: {
+    enabled: true,
+    plugins: [
+      {
+        name: 'mercurius-explain-graphiql-plugin',
+        props: { title: 'Mercurius explain graphiql plugin' },
+        umdUrl:
+          'https://unpkg.com/mercurius-explain-graphiql-plugin/dist/umd/index.js',
```

Comment:
maybe it is possibile to wrap this into a function that this library exposes

Reply:
yep it would be good addition to simplify the usage of the plugin!